cockatrice/src/dlg_add_set.cpp

```diff
@@ -0,0 +1,37 @@
+#include "dlg_add_set.h"
+
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+DlgAddSet::DlgAddSet(QWidget *parent, bool success) : QDialog(parent) {
+    status = new QLabel(this);
+    restart = new QLabel(this);
+
+    if (success) {
+        setWindowTitle(tr("Success"));
+        status->setText(QString("Sets/cards added to Cockatrice."));
+        restart->setText(QString("You must restart Cockatrice to use the new sets/cards."));
+    }
+    else {
+        setWindowTitle(tr("Failed"));
+        status->setText(QString("Sets/cards failed to import."));
```

Comment:
We should pipe through the failure reason into the dialog so that the user has an actionable outcome. Showing the error with no suggested action is likely to confuse people.